components/climate/ir_climate.rst

```diff
@@ -120,6 +120,17 @@ Additional configuration is available for this model
   * ``MODEL_DG11J1_3A``: Temperature range is from 18 to 32 (default)
   * ``MODEL_DG11J1_91``: Temperature range is from 16 to 30
 
+LG
+---------
+
+Additional configuration is available for this platform
+
+- **header_high** (*Optional*, int): time in us for the high part of the header for the LG protocol. Defaults to 8000
```

Comment:
please see other components how the time unit is specified, and expand the config for all 5 parameters

Reply:
Can you please point me to one component from which I can grab an example?

Reply:
Done, I think I'm now using the correct time standard for this.